src/plugins/explore/public/components/fields_selector/lib/get_details.ts

```diff
@@ -33,6 +33,7 @@ import { i18n } from '@osd/i18n';
 import { getFieldValueCounts } from './field_calculator';
 import { IndexPattern, IndexPatternField } from '../../../../../data/public';
 import { OpenSearchSearchHit } from '../../../types/doc_views_types';
+import { getServices } from '../../../application/legacy/discover/opensearch_dashboards_services';
 
 export function getDetails(
   field: IndexPatternField,
```

Comment:
Update to use the right type for me? Then we can modify that file instead of touching. Index pattern field